notebook.py

```
@@ -0,0 +1,53 @@
+# COMMAND ----------
+
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+# MAGIC %pip install pyyaml
+# MAGIC import yaml
+
+# COMMAND ----------
+
+from scr.nyctaxi.data_processor import DataProcessor
```

Ruff

Module level import not at top of file

(E402)